commands/backuplxc.go

```diff
@@ -204,6 +238,10 @@ func (c *backupLXCImplCommand) Run(ctx *cmd.Context) error {
 	if err := BackupLXCContainer(containerMachine, hostMachine, ctx.GetStdout()); err != nil {
 		return errors.Annotate(err, "backing up LXC container")
 	}
+	logger.Debugf("restarting LXC container %q", c.containerName)
+	if err := StartLXCContainer(containerMachine, hostMachine); err != nil {
```

Comment:
Why start the container again? Is this so that all the containers are in the same state after backing some of them up? If so, maybe it would be better to have the subsequent commands only work against stopped containers to reduce the risk of specifying `--match` for the backup command but forgetting it on the migrate command.

Reply:
The idea behind --match is so that HA workloads can be migrated one machine at a time. In that case, you'll:

 - backup the LXC machines one at a time, waiting for the workloads to come back online before proceeding to the next machine
 - likewise for migration to LXD

> If so, maybe it would be better to have the subsequent commands only work against stopped containers to reduce the risk of specifying --match for the backup command but forgetting it on the migrate command.

I don't understand. Are you suggesting backup and migrate individually? I guess we could do that, but there's no hard requirement that backup has been run at all. I'd prefer to keep them independent steps.

Reply:
Oh, I see. That makes sense - I think what I was saying would only make sense if for some reason you weren't migrating all of the containers, but obviously that doesn't work in the wider context of a 1.25 upgrade. I'm blaming cold medication!